exec/exec.c

```diff
@@ -31,7 +31,7 @@ int	exec(t_dyntree *root)
 		return (0);
 	if (exec_find_cmd(dynarr) == 0)
 	{
-		ft_puterror("Commande not found");
+		ft_puterror("Command not found");
```

Comment:
Yeah, well, that's just, like, your opinion, man.
https://www.youtube.com/watch?v=pWdd6_ZxX8c